pkg/hubble/parser/parser.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 )
 
+type Decoder interface {
```

Comment:
nit: missing docs on type and exported method

Reply:
Where can I update the docs?

Reply:
That's what I meant. All exported names should have comments describing their purpose. Thanks